gdk_pixbuf2/test/test-pixbuf.rb

```diff
@@ -164,4 +164,11 @@ def test_new_subpixbuf
       assert_equal(32, pixbuf.height)
     end
   end
+  
+  def test_fill!
+    filename = fixture_path("gnome-logo-icon.png")
+    pixbuf = GdkPixbuf::Pixbuf.new(:file => filename)
```

Comment:
Could you create two `GdkPixbuf::Pixbuf` instances? Because `#fill` is destructive method.